packages/core/src/blocks/FileBlockContent/fileBlockHelpers.ts

```diff
@@ -358,21 +353,21 @@ export const createResizeHandlesWrapper = (
 
     editor.updateBlock(block, {
       props: {
-        previewWidth: getWidth(),
+        previewWidth: parseInt(wrapper.style.width.replace("px", "")),
```

Comment:
Not at the moment, when the new width is calculated during resizing it goes straight into the style.  I did this intentionally because I felt it was redundant to also save to a variable, since it's the same information in 2 different places. But I can change this if you feel the parsing is too hacky.

Also in case you're wondering, `block.props.previewWidth` only gets updated when the mouse button is released, not while the user is dragging a resize handle, so it's not useful here.